db/seeds.rb

```diff
@@ -24,6 +24,13 @@
   { label: "Excellent", grade: 10, color: "00FF00", reading_group_id: reading_group.id }
 ])
 
+terrible = readings.first.id
+worrying = readings[2].id
+regular = readings[4].id
+satisfactory = readings[5].id
+good = readings[6].id
+excellent = readings.last.id
+
```

Comment:
This have only changed the location of the problem. Why don't we break the readings' and assign variables for each new created reading?